cardano-node/ChangeLog.md

```diff
@@ -1,6 +1,44 @@
 # Changelog for cardano-node
 
-## 1.31 -- October 2021
+## 1.32.0 -- November 2021
+
+### node changes
+
+- Miscellaneous documentation updates. (#3138, #3286, #3323, #3342, #3343,
+  #3348)
+- Various internal refactorings. (#3337, #3354, #3355)
+- Update a bunch of scripts to use the $NETWORK_MAGIC environment variable.
+  (#3148)
+- Integrate p2p networking functionality. (#3363)
+
+### consensus changes
+
+- Add per-miniprotocol byte limits, for things like how big a block arriving
+  from the network may be. (#3409)
+- Various documentation updates. (#3435, #3461, #3446)
+
+### network changes
+
+- Final PR adding p2p networking support. (#3467)
```

Comment:
It should be made clear that p2p is untested and disabled by default.

Reply:
```suggestion
- Early, unverified and unsupported p2p networking capabilities.  Do NOT use in production.  (#3467)
```